lib/flutter_vlc_player.dart

```diff
@@ -320,7 +320,7 @@ class VlcPlayerController {
   }
 
   Future<Uint8List> takeSnapshot() async {
-    var result = await _methodChannel.invokeMethod("getSnapshot");
+    var result = await _methodChannel.invokeMethod("getSnapshot", {'test': ''});
```

Comment:
Do we need to pass this 'test' data always or it can be removed?

Reply:
it was a quick fix, if no arguments then the invoke is ignored

Reply:
just change 'test' to 'getSnapShot' then

Reply:
@Antonimo can you make the change?